src/overlays/ovl_i6/fox_ve2.c

```diff
@@ -0,0 +1,4132 @@
+#include "global.h"
+
+f32 D_i6_801A7F40;
+f32 D_i6_801A7F44;
+f32 D_i6_801A7F48;
+f32 D_i6_801A7F4C;
+f32 D_i6_801A7F50;
+f32 D_i6_801A7F54;
+f32 D_i6_801A7F58;
+f32 D_i6_801A7F5C;
+f32 D_i6_801A7F60;
+f32 D_i6_801A7F64;
+f32 D_i6_801A7F68;
+f32 D_i6_801A7F6C;
+f32 D_i6_801A7F70;
+f32 D_i6_801A7F74;
+f32 D_i6_801A7F78;
+f32 D_i6_801A7F7C;
+Vec3f D_i6_801A7F80[100];
+f32 D_i6_801A8430;
+
+static f32* D_i6_801A6760[12] = {
+    D_C037FCC, D_C0380C0, D_C0381B4, D_C0382A8, D_C03839C, D_C038490,
+    D_C038584, D_C038678, D_C03876C, D_C038860, D_C038954, D_C038954,
+};
+
+void func_i6_80187530(Actor* actor) {
+    s32 index;
+
+    switch (actor->unk_048) {
+        case 0:
+            break;
+        case 1:
+            actor->unk_0B6 += (s32) actor->fwork[22];
+            if (actor->unk_0B6 >= Animation_GetFrameCount(&D_6014658)) {
+                actor->unk_0B6 = Animation_GetFrameCount(&D_6014658) - 1;
+            }
+            break;
+        case 2:
+            actor->unk_0B6 -= (s32) actor->fwork[22];
+            if (actor->unk_0B6 <= 0) {
+                actor->unk_0B6 = 0;
+            }
+            break;
+        default:
+            break;
+    }
+
+    index = actor->unk_0B6 / 18.0f;
+    actor->info.hitbox = SEGMENTED_TO_VIRTUAL(D_i6_801A6760[index]);
+    actor->obj.rot.z = D_i6_801A7F68;
+}
+
+s32 func_i6_8018767C(s32 limbIndex, Gfx** dList, Vec3f* pos, Vec3f* rot, void* data) {
+    if (limbIndex == 13) {
+        *dList = 0;
+    }
+    return 0;
+}
+
+extern Limb* D_6014844[];
+
+void func_i6_8018769C(Actor* actor) {
+    Vec3f sp28[20];
+
+    Animation_GetFrameData(&D_6014658, actor->unk_0B6, sp28);
+    Animation_DrawSkeleton(1, D_6014844, sp28, func_i6_8018767C, NULL, NULL, &gIdentityMatrix);
+}
+
+extern s32* D_C039290;
+extern s32* D_C03A290;
```

Comment:
```suggestion
extern u32 D_C039290[];
extern u32 D_C03A290[];
```
Textures should use unsigned types.